src/step_definitions/api.js

```diff
@@ -12,11 +12,20 @@ defineSupportCode(function({ When, Then }) {
     return service.fetch(method, endpoint).then(response => (fetchResult = response));
   });
 
+  When(/^I send "([^"]*)" request on "([^"]*)" endpoint with body:$/, function(method, endpoint, payload) {
+    // eslint-disable-next-line no-return-assign
```

Comment:
Is this necessary?

Reply:
Eslint doesn't like `(response => (fetchResult = response)`